Cargo.toml

```diff
@@ -22,17 +22,25 @@ curve25519-dalek = { version = "4.1.0", default-features = false, features = [
     "zeroize",
     "precomputed-tables",
     "legacy_compatibility",
+    "rand_core",
+    "serde",
```

Comment:
How is rand_core being added as a feature?  rand_core is not an ptional dependency

Reply:
I am not sure, does it have to be?

Reply:
rand_core is mandatory.  why is this feature being added?

Reply:
Because without it it does not compile, I don't know why.